packages/js-dpp/lib/test/fixtures/getIdentityFixture.js

```diff
@@ -5,12 +5,21 @@ const protocolVersion = require('../../version/protocolVersion');
 const Identity = require('../../identity/Identity');
 const IdentityPublicKey = require('../../identity/IdentityPublicKey');
 
-const id = generateRandomIdentifier();
+const randomIdentifier = generateRandomIdentifier();
 
 /**
+ * @param {Identifier} id
+ * @param {Identity} [IdentityClass]  - needed for usage in other scopes
+ * aside from dpp tests (platform-test-suite in karma)
+ * @param {IdentityPublicKey} [IdentityPublicKeyClass]  - needed for usage in other scopes
+ * aside from dpp tests (platform-test-suite in karma)
  * @return {Identity}
  */
-module.exports = function getIdentityFixture() {
+module.exports = function getIdentityFixture(
```

Comment:
Let's have separate fixtures for SDK. It should be simpler.

Reply:
Done